Index: framework/webapp/src/main/java/org/apache/ofbiz/webapp/control/LoginWorker.java
===================================================================
--- framework/webapp/src/main/java/org/apache/ofbiz/webapp/control/LoginWorker.java	(revision 78)
+++ framework/webapp/src/main/java/org/apache/ofbiz/webapp/control/LoginWorker.java	(working copy)
@@ -716,7 +716,7 @@
     }
 
     protected static String getAutoLoginCookieName(HttpServletRequest request) {
-        return UtilHttp.getApplicationName(request) + ".autoUserLoginId";
+        return UtilHttp.getApplicationName(request).replaceAll("/", ".") + ".autoUserLoginId";
     }
 
     public static String getAutoUserLoginId(HttpServletRequest request) {
